Zend/zend.h

```diff
@@ -137,12 +137,14 @@ struct _zend_class_entry {
 
 	int default_properties_count;
 	int default_static_members_count;
+	int constants_count;
 	zval *default_properties_table;
 	zval *default_static_members_table;
 	zval *static_members_table;
+	zval *constants_table;
 	HashTable function_table;
 	HashTable properties_info;
-	HashTable constants_table;
+	HashTable constants_info;
```

Comment:
Lucky to get a review out of you! Thanks for taking a careful look at this, and keeping php-src in top shape :)

If there is anything I can do code wise, just tell me. Would love to stay involved, but it is more efficient for you to work alone.

thanks